src/sentry/auth/authenticators/u2f.py

```diff
@@ -116,9 +122,35 @@ def activate(self, request):
 
         return ActivationChallengeResult(challenge=challenge)
 
-    def validate_response(self, request, challenge, response):
+    def validate_response(self, request, challenge, response, is_webauthn_signin_ff_enabled):
         try:
-            u2f.complete_authentication(challenge, response, self.u2f_facets)
+            if not is_webauthn_signin_ff_enabled:
+                u2f.complete_authentication(challenge, response, self.u2f_facets)
+                return True
+            # TODO change rp.id later when register is implemented
+            server = U2FFido2Server(
+                app_id=challenge["appId"],
+                rp={"id": challenge["appId"], "name": "Relying Party"},
```

Comment:
It seems like the "name" field should be something like "Sentry"? https://www.w3.org/TR/webauthn-2/#dictionary-pkcredentialentity